backend/mem/fileSystem.go

```diff
@@ -0,0 +1,49 @@
+package mem
+
+import (
+	"github.com/c2fo/vfs/v4"
+	"github.com/c2fo/vfs/v4/backend"
+	"github.com/c2fo/vfs/v4/utils"
+)
+
+//Scheme defines the filesystem type.
+const Scheme = "file"
+const name = "mem"
```

Comment:
name is should be a textual name of the filesystem type.
I'd suggest something like "In-Memory Filesystem"